dbOperations/removeBlacklist.js

```diff
@@ -0,0 +1,13 @@
+import { deleteDoc, doc } from "firebase/firestore";
+import { db } from "~/firebaseConfig";
+
+async function removeBlacklist(teamId, userId) {
+  try {
+    await deleteDoc(doc(db, "teams", teamId, "blacklist", userId));
+  } catch (e) {
+    console.log("Remove Blacklist Transaction failed: ", e);
```

Comment:
should this say transaction? 
```suggestion
    console.log("Remove User from Blacklist failed: ", e);
```